### DISCUSSION TITLE
Allow beatmaps to be reported from osu!lazer's beatmap browser

### DISCUSSION THREAD

**Discussion by Userlog2** [ppy/osu#29272] [Ideas]
Currently, unranked beatmaps can be reported on osu!web like this:

![image](https://github.com/user-attachments/assets/d17e9871-0be6-4489-bc8e-c4e8f24e6bd6)

There is no similar option in osu!lazer. The same area in lazer's beatmap browser looks like this (with no report option):

![image](https://github.com/user-attachments/assets/6388b9c6-3fa2-40c5-847b-b363eb6025e1)

This should be fixed; it's only a minor inconvenience (as it can be reported by clicking the link and opening the browser), but it's also an inconsistency.

**Comment by peppy** [comment]
You can click the link icon next to the beatmap's name to open the website for it, FWIW.

**Reply by Userlog2** [reply]
I know this. Edited original post to clarify.
